src/main/kotlin/com/ekino/oss/recaptcha/config/ReCaptchaConfiguration.kt

```diff
@@ -0,0 +1,74 @@
+package com.ekino.oss.recaptcha.config
+
+import com.ekino.oss.recaptcha.client.ReCaptchaClient
+import com.ekino.oss.recaptcha.service.ReCaptchaValidationService
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import mu.KotlinLogging
+import okhttp3.OkHttpClient
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication.Type.SERVLET
+import org.springframework.boot.context.properties.EnableConfigurationProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Import
+import org.springframework.http.HttpMethod
+import retrofit2.Retrofit
+import retrofit2.converter.jackson.JacksonConverterFactory
+
+private val logger = KotlinLogging.logger {}
+
+/**
+ * Configuration for reCaptcha filter.
+ * Available only for a web application.
```

Comment:
Small detail, but I would say something like `servlet application` or `servlet-based application`, since a reactive one is also a web application :)

Reply:
Yes you're right, corrected